Test/Install-WhiskeyNode.Tests.ps1

```diff
@@ -328,3 +430,24 @@ Describe 'Install-WhiskeyNode.when run in clean mode and Node is installed' {
         ThenNoError
     }
 }
+
+if($IsWindows)
+{
+    Describe 'Install-WhiskeyNode.when anti-virus locks file in uncompressed package' {
+        AfterEach { Reset }
+        It 'should still install Node.js' {
+            Init
+            GivenAntiVirusLockingFiles -AtLatestVersion -Seconds 13
+            ThenNodeInstalled -AtLatestVersion -NodeZipFileCheck
+        }
+
+        It 'should fail' {

```

Comment:
Each `Describe` block should only have one `It` block. It makes running single tests easier. Put the second `It` block in its own `Describe` block:

```powershell

Describe 'Install-WhiskeyNode.when anti-virus locks the file too long' {
    AfterEach { Reset }
}
```